src/public/js/video-tagging/optional-tags.html

```diff
@@ -113,27 +114,35 @@
             Polymer.dom(this.$.buttonsContainer).appendChild(btn);  
         }
 
-        window.addEventListener("keyup", (e) => {
+        if(videotagging.hotkeysAssigned){
+            window.removeEventListener("keyup", this.assignHotkeys);
```

Comment:
you could avoid the use of `self = ...` if you do a `.bind(this)` on the event handler.